reactivexcomponent/reactivexcomponent/configuration/api_configuration.py

```diff
@@ -0,0 +1,96 @@
+""" API Configuartion """
+
+from lxml import etree
+
+EVENT_TYPE = {"Update": "UPDATE", "Error": "ERROR"}
+
+KINDS = {"Snapshot": 1, "Private": 2, "Public": 3}
+
+COMMANDS = {"hb": "hb", "update": "update", "subscribe": "subscribe",
+            "get_xc_api": "getXcApi", "get_xc_api_list": "getXcApiList", "get_model": "getModel"}
+
+NAME_SPACE = {'xmlns': 'http://xcomponent.com/DeploymentConfig.xsd'}
+
+
+def format_fsharp_field(value):
+    return {"Case": "Some", "Fields": [value]}
+
+
+class APIConfiguration:
+
+    def __init__(self, file):
+        self.file = file
+        self.root = None
+
+    def get_xml_content(self):
```

Comment:
Rename to something as process xml file. This method doesn't return anything. It shouldn't be called "get"